velox/exec/HashBuild.cpp

```diff
@@ -1107,13 +1111,14 @@ void HashBuild::reclaim(
 
   // NOTE: a hash build operator is reclaimable if it is in the middle of table
   // build processing and is not under non-reclaimable execution section.
```

Comment:
Shall we move the comments to the top of or inside nonReclaimableState() method?